addons/controllers/cpi/vspherecpiconfig_utils.go

```diff
@@ -120,9 +120,17 @@ func (r *VSphereCPIConfigReconciler) mapCPIConfigToDataValuesNonParavirtual( //
 
 	d.Region = tryParseString(d.Region, c.Region)
 	d.Zone = tryParseString(d.Zone, c.Zone)
+
```

Comment:
Hi fang, could you remove this file's change? Use this PR to only remove the default value. But we should still assume the default value of insecureFlag should be explicitly passed from Cluster object

Reply:
It's expected that insecureFlag is deprected and will treat empty tlsThumbprint as insecure